TeamSnapSDK/SDK/DataTypes/TSDKTslPhotos.m

```diff
@@ -29,6 +29,7 @@ +(nonnull TSDKBackgroundUploadProgressMonitorDelegate *)uploadTSLImageURL:(nonnu
     TSDKBackgroundUploadProgressMonitorDelegate *backgroundUploadDelegate = [[TSDKBackgroundUploadProgressMonitorDelegate alloc] initWithProgressBlock:progressBlock];
     
     [TSDKDataRequest requestJSONObjectsForPath:url sendDataDictionary:nil method:@"GET" configuration:[TSDKRequestConfiguration requestConfigurationWithForceReload:YES] withCompletion:^(BOOL success, BOOL complete, id  _Nullable objects, NSError * _Nullable error) {
+        if(success) {
         TSDKCollectionJSON *collection = [[TSDKCollectionJSON alloc] initWithJSON:objects];
```

Comment:
These lines should be indented for readability.

:+1: Code 
:+1: Tests passed.